example/src/main.tsx

```diff
@@ -1,75 +1,179 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
-import { URLModal, openModal } from '../../';
-import Modal1 from './Modal1';
-import Modal2 from './Modal2';
 import '../../styles/index.css';
+import './style.css';
+import clipboardCopy from 'clipboard-copy';
+import { useState } from 'react';
+import { Code } from './components/Code';
+import {
+  dynamicImportedModal,
+  standardModalCode,
+  withCustomWrapper,
+  withModal,
+  withParams,
+} from './code';
+import {
+  CustomWrapper,
+  StandardModal,
+  WithModal,
+  WithParams,
+} from './components/Examples';
+
+enum PackageManagers {
+  NPM = 'npm',
+  YARN = 'yarn',
+  PNPM = 'pnpm',
+}
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ');
+}
+
+const getText = (packageManager: PackageManagers) => {
+  const packageName = 'react-url-modal';
+  switch (packageManager) {
+    case 'npm':
+      return `npm install ${packageName}`;
+    case 'pnpm':
+      return `pnpm install ${packageName}`;
+    case 'yarn':
+      return `yarn add ${packageName}`;
+  }
+};
 
 const App = () => {
+  const [packageManager, setPackageManager] = useState<PackageManagers>('npm');
```

Comment:
Here is the TS error

Reply:
I guess this need to be `PackageManagers.NPM` or changing PackageMangers to `type PackageManagers = 'npm' | 'pnpm' | 'yarn'`

Reply:
You are super right, I keep forgetting you can use TS enums as actual values. Ended up making use of `PackageManagers.NPM` and its way cleaner now and has no random strings

Thank you

Reply:
Happy I could help ☺️ 

Looks pretty nice 🎉

Reply:
Danke :D